svg-convert.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
```

Comment:
I'm never sure whether to add copyright notices when contributing to someone else's project. Done. :)